bentoml/deployment/aws_lambda/utils.py

```diff
@@ -247,3 +263,135 @@ def init_sam_project(
     logger.debug('Removing unnecessary files to free up space')
     for api_name in api_names:
         cleanup_build_files(sam_project_path, api_name)
+
+
+def _sum_total_directory_size(directory):
+    dir_size = sum(
+        f.stat().st_size for f in Path(directory).glob('**/*') if f.is_file()
+    )
+    return dir_size
+
+
+def is_build_function_size_under_lambda_limit(build_directory):
+    dir_size = _sum_total_directory_size(build_directory)
+    logger.debug('Directory %s size is %d', build_directory, dir_size)
+    return False if dir_size > LAMBDA_FUNCTION_LIMIT else True
+
+
+def is_build_function_size_over_max_lambda_size(build_directory):
+    dir_size = _sum_total_directory_size(build_directory)
+    return True if dir_size > LAMBDA_FUNCTION_MAX_LIMIT else False
+
+
+def reduce_lambda_function_size(
```

Comment:
rename to `reduce_lambda_build_directory_size` and move build_directory to the first parameter?